source/pkcore/about.rst

```diff
@@ -37,10 +37,12 @@ Elements
    Similar to Earthbenders, they employ both defensive and offensive
    strategies to accomplish this.
 
--  Chiblocking is the anti-bending fighting style. Chiblockers use critically
-   placed kicks and punches to block chi paths in bender's bodies, doing
-   anything from disabling their bending to temporarily paralyzing
-   them.
+-  Chi is the life energy that flows through everything. Humans, along
```

Comment:
If you are going to introduce TLA/TLOK Lore, you better do it everywhere. You can rewrite this so that it flows straight to the point instead of going into the tangent of how benders came to be.

Reply:
no it's definitely fine

Reply:
No it definitely isn't, it looks wayyy to different from the rest of them.

Reply:
I agree with @Varhagna and we need to stay consistant